lab-isaac/lib/read-file.js

```diff
@@ -0,0 +1,30 @@
+'use-strict';
+
+
+const vegies = module.exports = {};
+const fs = require('fs');
+
+vegies.read = (callback) => {
+  let first, second, third;
+
+  fs.readFile(`${__dirname}/../assets/one.txt`, (err, data) => {
+    if(err) console.error(err);
+    first = data.toString('hex', 0, 8);
+
```

Comment:
Part of the assignment was to console log each file as its processed so you know they are being read in the correct order.